app/models/petition.rb

```diff
@@ -1152,7 +1152,7 @@ def sponsor_count
 
   def state_for_publishing(time)
     if open_at
-      if closed_at > time
+      if closed_at && closed_at > time
```

Comment:
Is this right? If `open_at` is set and `closed_at` isn't then it will put it into the closed state